Packs/Core/TestPlaybooks/Test Playbook - Possible External RDP Brute-Force - Set Verdict.yml

```
@@ -0,0 +1,1456 @@
+      playbookName: Possible External RDP Brute-Force - Set Verdict
+      type: playbook
+      iscommand: false
+      brand: ""
+    nexttasks:
+      '#none#':
+      - "10"
+    scriptarguments:
+      HuntResults:
+        simple: "True"
+      IpReputation:
+        simple: "2"
+      RelatedAlerts:
+        simple: "True"
+      RelatedCampaign:
+        simple: "True"
+      RiskyHost:
+        complex:
+          root: Core.RiskyHost
+          filters:
+          - - operator: isEqualString
+              left:
+                value:
+                  simple: Core.RiskyHost.risk_level
+                iscontext: true
+              right:
+                value:
+                  simple: MED
+              ignorecase: true
+            - operator: isEqualString
+              left:
+                value:
+                  simple: Core.RiskyHost.risk_level
+                iscontext: true
+              right:
+                value:
+                  simple: HIGH
+              ignorecase: true
+          accessor: id
+      RiskyUser:
+        complex:
+          root: Core.RiskyUser
+          filters:
+          - - operator: isEqualString
+              left:
+                value:
+                  simple: Core.RiskyUser.risk_level
+                iscontext: true
+              right:
+                value:
+                  simple: MED
+              ignorecase: true
+            - operator: isEqualString
+              left:
+                value:
+                  simple: Core.RiskyUser.risk_level
+                iscontext: true
+              right:
+                value:
+                  simple: HIGH
+              ignorecase: true
+          accessor: id
+      UnusualGeoLocation:
+        simple: "True"
+    separatecontext: false
+    continueonerrortype: ""
+    loop:
+      iscommand: false
+      exitCondition: ""
+      wait: 1
+      max: 100
+    view: |-
+      {
+        "position": {
+          "x": 450,
+          "y": 330
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "311":
+    id: "311"
+    taskid: 1ac28b39-0163-495b-8952-c1cfcc76c024
+    type: regular
+    task:
+      id: 1ac28b39-0163-495b-8952-c1cfcc76c024
+      version: -1
+      name: Set 'RiskyHost' To Context
+      description: Set multiple keys/values to the context.
+      scriptName: SetMultipleValues
+      type: regular
+      iscommand: false
+      brand: ""
+    nexttasks:
+      '#none#':
+      - "309"
+    scriptarguments:
+      keys:
+        simple: risk_level,id
+      parent:
+        simple: Core.RiskyHost
+      values:
+        simple: MED,1111
```

Change this to be different from the endpoint
```suggestion
        simple: HIGH,2222
```

---

changed